### DISCUSSION TITLE
Pending review for  Klip Wallet (Project ID: 2914842ae8a)

### DISCUSSION THREAD

**Discussion by aoifehan** [WalletConnect/walletconnect-monorepo#2291] [Developer Support]
We've been working on setting up our project (ID: 2914842ae8a) for a while now, but it still seems to be pending review. We submitted it for review three weeks ago, and our domain was verified 10 days ago, but we have yet to receive any updates. We've also reached out to the Discord developer support channel, but unfortunately, we have not received a response. Could you please look into this and help our project get reviewed as soon as possible? Thanks!

**Comment by finessevanes** [comment]
Hey, I am currently in the US and trying to download the wallet. However, I am encountering an error message stating that the wallet is not available in my country. Can you please provide information on which countries are excluded from accessing the wallet?

**Comment by aoifehan** [comment]
Hey @finessevanes ! Thanks for looking into this. Currently, Klip is only available for download in the South Korean market. Would it be possible for you to set your app store to the South Korean region?

**Comment by aoifehan** [comment]
Hey @finessevanes ! It's been a while. I'm hoping to see if there has been an update :)

**Reply by finessevanes** [reply]
Hey - can I use a VPN?

**Reply by brian-eykim** [reply]
@finessevanes hi, I'm a coworker of @aoifehan .
It's totally up to you to use a VPN.

**Reply by finessevanes** [reply]
I switched my VPN to South Korea but was unable to download the wallet.

**Reply by aoifehan** [reply]
Hey there @finessevanes did you switch your App Store to Korean region? The app should be available regardless of the VPN setting as long as you have the app store set up to Korean market. 

Are you simply checking to ensure that the app is functioning correctly, or are you planning to sign up and explore its features as well? Keep in mind that after downloading the app, you will need a KakaoTalk account for identity verification and the sign-up process. However, please note that this KakaoTalk verification process is only available for users within the country.

**Reply by aoifehan** [reply]
Hey @finessevanes , we recently discussed internally how to get you to test the app. If you could kindly share your Apple Store email with us, we'll be able to add you to TestFlight. We also have a dedicated test KakaoTalk account that you can use for logging in. Once you provide us with your contact email, we'll send you all the necessary information via email.

**Reply by finessevanes** [reply]
In order for the wallet to be approved, it needs to be in prod, we don't accept TestFlight submissions. We can still approve you on Android, just be sure the togggle the right options. I just checked your application and it looks like it was rejected with notes.

I'm going to close this discussion as further communication can be done via email with our designated Explorer team.

**Reply by aoifehan** [reply]
Hey @finessevanes  We just received a rejection from the project registry with the following message.

> Please troubleshoot, retest and submit your changes so that we can review and approve the request for you: Indicated v2 support for WalletConnect Sign SDK

Does this mean that we need to support sign v2 in order to get submission through?
